lib/components/dashboard/Metrics/TableWrapper.tsx

```diff
@@ -48,23 +62,36 @@ export async function UsageTable({
 	if (!usageQuery.success) return <p>Bad request</p>
 	if (!user) redirect('/')
 
+	const team = await prisma.team.findUnique({
+		where: {slug: teamSlug},
+		select: {
+			id: true
+		}
+	})
+
+	if (!team) return <p>Team not found</p>
+
 	const pageSize = 5
 	const pageNum = usageQuery.data.p
 
-	const usageFilter = {
-		project: {user: {id: user.id}},
-		action: {contains: usageQuery.data.q}
-	}
+	// const usageFilter = {
+	// 	project: {user: {id: user.id}},
+	// 	agent: {contains: usageQuery.data.q}
+	// }
```

Comment:
Commented code

Reply:
Commented since search can be added later, not supported atm since idk which column we actually would use search for.